ecl_geometry/include/ecl/CMakeLists.txt

```diff
@@ -3,12 +3,13 @@
 ###############################################################################
 
 add_subdirectory(geometry)
+add_subdirectory(odometry)
```

Comment:
This is a little bit mixed up...should stay as part of geometry so it's easy for the user to discover. i.e. something like:

* `include/ecl/geometry/odometry.hpp`
* `include/ecl/geometry/odometry_typedefs.hpp`
* `include/ecl/geometry/odometry_helpers.hpp`